tool/gc_decode.cpp

```diff
@@ -0,0 +1,62 @@
+#include "IRsend.h"
+#include "IRsend_test.h"
```

Comment:
Per other comments. Lets use the original file.

<nitpick> Also, the ordering of the headers I think needs to be different per styleguide.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes